core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/UnixProcessManager.java

```diff
@@ -90,7 +90,7 @@ public void kill(Process process) {
     if (C_LIBRARY != null) {
       killTree(getPid(process));
     } else {
-      throw new IllegalStateException("Can't kill process. Not unix system?");
+      throw new IllegalStateException("Cannot terminate process. Not a UNIX system?");
```

Comment:
Is this about terminating, or sending a specific signal (there are both TERM and KILL signals)?

Reply:
Not sure. I'll revert back to `kill` to avoid possibly mangling the meaning.